### PR TITLE
speedtest-cli: add page

### LINKED ISSUE #1787: page request: speedtest-cli
Test down/uplink connection via Speedtest.net servers with shareable image (`--share`)

Reference: https://github.com/sivel/speedtest-cli#usage

### PR DESCRIPTION
**Pull request by Scarsz**
Fixes #1787 

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by Scarsz** [comment]
Neat little bot you guys have here. :-)

File: pages/common/speedtest-cli.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Run a speed test:
+
+`speedtest-cli`
+
+- Run a speed test and generate a sharable result picture:
+
+`speedtest-cli --share`
+
+- Print a list of all speedtest.net servers, sorted by distance, to file:
```
Comment (agnivade) [inline pages/common/speedtest-cli.md]: Let's change this to print to console. I don't see a specific reason to redirect to file.

File: pages/common/speedtest-cli.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`speedtest-cli --list > speedtest_servers.txt`
+
+- Run a speed test to the given speedtest.net server number:
```
Comment (agnivade) [inline pages/common/speedtest-cli.md]: number => id

File: pages/common/speedtest-cli.md (hunk 1 above)
Comment (Scarsz) [reply pages/common/speedtest-cli.md]: The output is over 7000 lines long (7029 to be specific).

Comment (agnivade) [reply pages/common/speedtest-cli.md]: Haha, okay then.

**Comment by sbrl** [comment]
Indeed we do! If you're curious to read its source code, it's available over in [tldr-pages/tldr-bot](https://github.com/tldr-pages/tldr-bot/).

And this looks ok to me! Thanks for the page, @Scarsz 😺

**Pull request merged**